Sources/ArcGISToolkit/Extensions/ArcGIS/AttachmentHelpers/AttachmentsFeatureElement.swift

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import Foundation
+
+/// Indicates how to display the attachments. If `list` is specified, attachments show as links. If `preview` is specified, attachments expand to the width of the view. Setting the value to `auto` allows applications to choose the most suitable default experience for their application.
+public enum AttachmentsFeatureElementDisplayType {
+    /// Show attachments as links.
+    case list
+    /// Attachments expand to the width of the view.
+    case preview
+    /// Allows applications to choose the most suitable default experience.
+    case auto
+}
+
+/// Common properties for elements which display feature attachments.
+public protocol AttachmentsFeatureElement {
+    /// Indicates how to display the attachments.
+    /// If `list` is specified, attachments show as links. If `preview` is specified, attachments expand to the width of the view. Setting the value to `auto` allows applications to choose the most suitable default experience.
```

Comment:
Just splits into multiple lines.
```suggestion
    /// If `list` is specified, attachments show as links.
    /// If `preview` is specified, attachments expand to the width of the view.
    /// Setting the value to `auto` allows applications to choose the most suitable default experience.
```

I think we could also just rely on the doc provided by the enum itself.
```suggestion
```